components/Switch/Switch.stories.tsx

```diff
@@ -29,3 +32,85 @@ Large.args = { size: '2' };
 export const Disabled = Template.bind({});
 
 Disabled.args = { disabled: true };
+
+export const Labelled: ComponentStory<typeof SwitchForStory> = ({ id, ...args }) => (
+  <>
+    <Flex align="center">
+      <Label variant="contrast" htmlFor={id}>label</Label>
+      <SwitchForStory id={id} {...args} />
+    </Flex>
+  </>
```

Comment:
No need for the Fragments here

```suggestion
    <Flex align="center">
      <Label variant="contrast" htmlFor={id}>label</Label>
      <SwitchForStory id={id} {...args} />
    </Flex>
```